app/conf/als-example.conf

```diff
@@ -62,4 +62,19 @@ oryx {
       port = 8080
     }
   }
+  ml {
+    eval {
+      candidates = 1
+      parallelism = 1
+    }
+  }
+  als {
+    iterations = 20
+    implicit = true
+    hyperparams {
+      features = [ 10, 10 ]
```

Comment:
I think this is fine to add, but it's testing the same values here several times which doesn't help. You could also show trying multiple candidates above or else this also won't do anything. Finally, nit pick but a space would be good before the closing bracket on the next line.